apps/src/blockly/customBlocks/googleBlockly/mutators/functionMutatorHelpers.js

```diff
@@ -7,10 +7,13 @@ import msg from '@cdo/locale';
 export function setBlockDescription(block, description) {
   if (Blockly.useModalFunctionEditor) {
     block
-      .appendDummyInput('DESCRIPTION_ROW')
+      .appendEndRowInput('DESCRIPTION_ROW')
       .appendField(`${msg.description()} `, 'DESCRIPTION_LABEL')
-      .appendField(new Blockly.FieldTextInput(description), 'DESCRIPTION');
-    block.moveInputBefore('DESCRIPTION_ROW', 'STACK');
+      .appendField(new Blockly.FieldMultilineInput(description), 'DESCRIPTION');
+    const inputToPrecede = block.getInput('flyout_input')
+      ? 'flyout_input'
+      : 'STACK';
+    block.moveInputBefore('DESCRIPTION_ROW', inputToPrecede);
```

Comment:
why did this need to change?

Reply:
Good catch; apologies for missing this in the PR write up! I did this to better align to CDO Blockly's modal editor. ie. the flyout is below the description field (and closer to the definition blocks)
![image](https://github.com/code-dot-org/code-dot-org/assets/43474485/f067ab84-4545-462c-8843-98466b891fd8)

I brought this up on Slack on 12/14: https://codedotorg.slack.com/archives/C05HZFH7BED/p1702585446512979